CloudFormationWindows/lansa-master-win.cfn.template

```diff
@@ -1167,7 +1167,7 @@
     },
     "CustomVariable": {
       "TemplateRoot": {
-        "Value": "https://s3-ap-southeast-2.amazonaws.com/lansa/templates/"
+        "Value": "https://s3-ap-southeast-2.amazonaws.com/lansa/templates/support/scalable"
       },
```

Comment:
this is not correct. Revert it. What issue were you trying to solve?

Reply:
removed support/scalable from the TemplateRoot path